src/valid/interface.rs

```diff
@@ -353,7 +355,6 @@ impl VaryingContext<'_> {
                 .map_err(|e| e.with_span_context(span_context)),
             None => {
                 match self.types[ty].inner {
-                    //TODO: check the member types
```

Comment:
Thanks for removing this. It's covered in `validate_impl` now, right?

Reply:
Yep